app/__init__.py

```diff
@@ -68,4 +68,13 @@ def create_app(config_name):
     from bulk import bulk as bulk_blueprint
     app.register_blueprint(bulk_blueprint, url_prefix='/bulk')
 
+    from descriptor import descriptor as descriptor_blueprint
```

Comment:
Yeah, I see what you mean - it's more natural to think that one descriptor describes one specific thing. But between `descriptor` and `descriptor_response` I would still choose the former, because it's also unnatural to think about something responding to a description. I'd say let's leave it as it is unless we find a more intuitive alternative. The inconsistency in the meaning doesn't seem to be very big to me in this case, especially if we document everything clearly.

Do you think it's a big concern that an eponymous model exists? It doesn't seem so to me, since both are meant to mean the same thing.